mm2src/mm2_main/src/lp_ordermatch/simple_market_maker.rs

```diff
@@ -518,7 +516,22 @@ async fn update_single_order(
     let resp = update_maker_order(ctx, req)
         .await
         .map_to_mm(OrderProcessingError::OrderUpdateError)?;
-    info!("Successfully update order for {} - uuid: {}", key_trade_pair, resp.uuid);
+    if is_max {
+        info!(
+            "Successfully update order for {key_trade_pair} - uuid: {} - rate: ({:.8} {key_trade_pair}) - max volume",
+            resp.uuid,
+            calculated_price.to_decimal()
+        );
+    } else {
```

Comment:
Could you please refactor this as
```rust
let vol_info = if is_max {
    "max volume".to_string()
} else {
    format!("volume: {:.8}", volume.to_decimal())
};
info!("Successfully update order for {key_trade_pair} - uuid: {} - rate: ({:.8} {key_trade_pair}) - {vol_info}", resp.uuid, calculated_price.to_decimal());
```

Reply:
is this really needed? done anyways

Reply:
Do you think the changed version looks more clear and readable?
It also helped to to shorten by 4 lines